Assets/Scripts/StandardSamples/Discord/DiscordManager.cs

```diff
@@ -41,6 +41,52 @@ namespace PlayEveryWare.EpicOnlineServices.Samples.Discord
     [DisallowMultipleComponent]
     public class DiscordManager : MonoBehaviour
     {
+        public void StartConnectLogin(EOSManager.OnConnectLoginCallback onLoginCallback)
+        {
+#if !DISABLEDISCORD
```

Comment:
I think that this shouldn't be surrounded by this ifdef. Users can only call the `public` version here, which will call the `private` version, and the `private` version has a log about Discord not being supported. I think that when the `public` version is called, it should just call the inner `private` version, so that it will log that error out.

Reply:
You are totally right, fixing that now.